src/fides/api/ctl/routes/crud.py

```diff
@@ -83,7 +84,8 @@ async def validate_data_categories(
         status_code=status.HTTP_201_CREATED,
         dependencies=[
             Security(
-                verify_oauth_client_cli, scopes=[scope_registry.CLI_OBJECTS_CREATE]
+                verify_oauth_client_cli,
+                scopes=[f"{CLI_SCOPE_PREFIX_MAPPING[model_type]}:{CREATE}"],
```

Comment:
Thanks for this Dawn, thinking deeper into it I just want to flag that creating scopes inside endpoints is a break from the convention of pre-defining them. Although this is useful it strikes me as an anti-pattern specifically for scopes since:
- there'll be no one place where we can see at a glance exactly what scopes are supported by the application, and
- no single place we'll be able to import them all from either (which is useful when creating the root client for instance)

Reply:
Yes I pondered this too, there's pros and cons about defining these crud endpoints in a loop and these scopes are one of the cons. To push back a bit though, the scope registry still has all the scopes in a single place, and the root user is also given the scopes you see here.